File: Baekjoon/BaaarkingDog/0x02_basic/2445.py
# 2445: 별 찍기 - 8

N = int(input())
step = 1
i = 1

while (i > 0):
  if (i == N): step = -1

  print("*" * i, end="")
  print(" " * ((N-i) * 2), end="")
  print("*" * i)

  i += step

File: Baekjoon/BaaarkingDog/0x02_basic/2445.cpp
// 2445: 별 찍기 - 8

#include <iostream>

int main() {
  int N, step = 1;
  std::cin >> N;

  for (int i=1; i>0; i+=step) {
    if (i == N) step = -1;

    // 좌측 별
    for (int j=0; j<i; j++) std::cout << '*';
    // 중앙 공백
    for (int j=0; j<(N-i)*2; j++) std::cout << ' ';
    // 우측 별
    for (int j=0; j<i; j++) std::cout << '*';

    std::cout << '\n';
  }

  return 0;
}
